src/app/projects/projects-list/projects-list.component.html

```diff
@@ -83,6 +83,7 @@
         <th class="vf-table__heading" scope="col">Project</th>
         <th class="vf-table__heading" scope="col">Organ</th>
         <th class="vf-table__heading" scope="col">Technology</th>
+        <th class="vf-table__heading" scope="col">Cell count</th>
```

Comment:
I know it is picky, but you were using `Cell Count` at Ingest UI.
Could you use the same here too, please?

Reply:
Then "Data location" should also be changed

Reply:
and "Date added"? Actually, i think the small case for the second word is preferred. I'll ask Gabs later.

Reply:
@gabsie Do you have suggestions on how the casing should be for the 2 word columns?

Reply:
yes - small c in count ;)

Reply:
okies, thanks!